src/components/EnterScreen.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div class="enter-screen">
+    <img
+      class="logo"
+      src="/icons/logo.svg"
+    >
+    <div class="buttons-container">
+      <ButtonText
+        text="sign up"
+        @click="_openSidebar('sign up')"
```

Comment:
I think that it's better not to use spaces in some properties which are for inside use.
What do you think?